DEVELOPMENT.md

```diff
@@ -72,6 +72,11 @@ Install these tools:
     cmd/operator/generate.sh
     ```
 
+1. Install a database:
+   ```bash
+   hack/install-database.sh
```

Comment:
Can't we put these in an `integrations/database/postgres` directory?

Reply:
I don't have any arguments not to move it, @rh-hemartin, do you?